### PR TITLE
feat(cmd): add new flag to allow granular control over level of each logging module

### PR DESCRIPTION
**Pull request by Wondertan**
No issue here, just as simple flag and lowering verbosity of some annoying logs in our deps

### CODE REVIEW

**Review by vgonkivs — APPROVED** [review]

File: cmd/flags_misc.go [hunk 1]
```diff
@@ -14,8 +15,9 @@ import (
 )
 
 var (
-	logLevelFlag = "log.level"
-	pprofFlag    = "pprof"
+	logLevelFlag       = "log.level"
+	logLevelModuleFlag = "log.level.module"
```
Comment (renaynay) [inline cmd/flags_misc.go:19]: can't this just be `log.module`

File: cmd/flags_misc.go [hunk 2]
```diff
@@ -50,6 +58,18 @@ func ParseMiscFlags(cmd *cobra.Command) error {
 		logs.SetAllLoggers(level)
 	}
 
+	logModules, err := cmd.Flags().GetStringSlice(logLevelModuleFlag)
+	if err != nil {
+		return err
+	}
+	for _, ll := range logModules {
+		params := strings.Split(ll, ":")
+		err := logging.SetLogLevel(params[0], params[1])
```
Comment (renaynay) [inline cmd/flags_misc.go:67]: This could panic if user inputs log module the wrong way (doesn't use `:` or something). Could check that len(params) == 2.

Comment (Wondertan) [reply cmd/flags_misc.go:67]: Yeah, true.

Comment (Wondertan) [reply cmd/flags_misc.go:71]: Fixed

File: cmd/flags_misc.go (hunk 1 above)
Comment (renaynay) [reply cmd/flags_misc.go:19]: lets just leave as is. I never seen a flag with two `.` in it 😂  

If we have complaints, we can simplify.

**Pull request merged**